package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "swift-doc-assets",
+  "version": "1.0.0",
+  "browserslist": [
+    "> 1%",
+    "not dead"
+  ],
+  "description": "Front-end assets for swift-doc",
+  "main": "Assets/index.css",
+  "scripts": {
+    "clean": "rm -rf .parcel-cache",
+    "watch": "parcel watch Sources/web/css/**/*.css",
+    "build": "parcel build Sources/web/css/styles.css"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "autoprefixer": "^9.7.5",
+    "parcel": "^2.0.0-alpha.3.2",
+    "postcss-preset-env": "^6.7.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SwiftDocOrg/swift-doc.git"
+  },
+  "keywords": [],
+  "contributors": [
+    "Reda Lemeden <hello@redalemeden.com> (http://redalemeden.com)"
+  ],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/SwiftDocOrg/swift-doc/issues"
+  },
+  "homepage": "https://github.com/SwiftDocOrg/swift-doc#readme"
+}
```

Comment:
How much of the default generated package.json metadata can we scrap? Can we get away with just `browserslist`, `dependencies`, `devDependencies`, and `scripts`?

Reply:
Yep, that should work.